src/npcmove.cpp

```diff
@@ -1231,48 +1234,45 @@ double npc::confidence_mult() const
 
     switch( rules.aim ) {
         case AIM_WHEN_CONVENIENT:
-            return emergency() ? 1.0f : 0.75f;
+            return emergency() ? 1.5f : 1.0f;
```

Comment:
Btw. why is this function returning `float` values when it's declared as return `double` values. The `f` at the end of the number is completely pointless, the compiler will effectively remove it (at compile or at runtime).